testutil/ciao-down/workload.go

```diff
@@ -76,6 +81,40 @@ func (wkld *workload) save(ws *workspace) error {
 }
 
 func loadWorkloadData(ws *workspace, workloadName string) ([]byte, error) {
```

Comment:
This function and the function that call it need to take a context parameter.  The context parameter will be passed in to createWorkload from the createFlags function.  It must be passed to the context.WithTimeout call below.  If you do this, hitting CTRL-C at the console will cancel the context you pass to getFile which is what we want.